content/zh-cn/docs/concepts/workloads/pods/_index.md

```diff
@@ -538,30 +537,55 @@ Pod 中的容器所看到的系统主机名与为 Pod 配置的 `name` 属性值
 <!--
 ## Privileged mode for containers
 
-In Linux, any container in a Pod can enable privileged mode using the `privileged` (Linux) flag on the [security context](/docs/tasks/configure-pod-container/security-context/) of the container spec. This is useful for containers that want to use operating system administrative capabilities such as manipulating the network stack or accessing hardware devices.
+{{< note >}}
+Your {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}} must support the concept of a privileged container for this setting to be relevant.
+{{< /note >}}
 
-If your cluster has the `WindowsHostProcessContainers` feature enabled, you can create a [Windows HostProcess pod](/docs/tasks/configure-pod-container/create-hostprocess-pod) by setting the `windowsOptions.hostProcess` flag on the security context of the pod spec. All containers in these pods must run as Windows HostProcess containers. HostProcess pods run directly on the host and can also be used to perform administrative tasks as is done with Linux privileged containers.
--->
-## 容器的特权模式     {#privileged-mode-for-containers}
+Any container in a pod can run in privileged mode to use operating system administrative capabilities
+that would otherwise be inaccessible. This is available for both Windows and Linux.
```

Comment:
The translation paragraph is too long for the reviewer, please change back to the previous segmentation method.

see [the guide](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#%E6%AE%B5%E8%90%BD%E5%88%92%E5%88%86) for more details.